### PR TITLE
minor cleanup and code refactor in web_peer_connection.cpp

### PR DESCRIPTION
**Pull request by aldenml**

### CODE REVIEW

File: src/web_peer_connection.cpp [hunk 1]
```diff
@@ -514,7 +514,7 @@ namespace {
 				ec = errors::no_content_length;
 			}
 		}
-		return std::tuple<std::int64_t, std::int64_t>(range_start, range_end);
+		return {range_start, range_end};
```
Comment (arvidn) [inline src/web_peer_connection.cpp]: this is actually not supported by C++11. It has been fixed in later versions, but I'm trying to stick to C++11 for now. libc++ supports it as an extension.

Comment (arvidn) [reply src/web_peer_connection.cpp]: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4387.html

**Comment by arvidn** [comment]
lgtm apart from comment

File: src/web_peer_connection.cpp (hunk 1 above)
Comment (aldenml) [reply src/web_peer_connection.cpp]: right, now I remember that, read about it a long time ago

**Comment by arvidn** [comment]
lgtm

**Pull request merged**